OpenRA.Mods.Common/Widgets/Logic/GlobalChatLogic.cs

```diff
@@ -27,10 +27,13 @@ class GlobalChatLogic : ChromeLogic
 		public GlobalChatLogic(Widget widget)
 		{
 			historyPanel = widget.Get<ScrollPanelWidget>("HISTORY_PANEL");
-			historyTemplate = historyPanel.Get<LabelWidget>("HISTORY_TEMPLATE");
+			chatTemplate = historyPanel.Get<ContainerWidget>("CHAT_TEMPLATE");
 			nicknamePanel = widget.Get<ScrollPanelWidget>("NICKNAME_PANEL");
 			nicknameTemplate = nicknamePanel.Get("NICKNAME_TEMPLATE");
 
+			var textLabel = chatTemplate.Get<LabelWidget>("TEXT");
+			textLabel.GetColor = () => ChromeMetrics.Get<Color>("GlobalChatTextColor");
```

Comment:
Assign `ChromeMetrics.Get<Color>("GlobalChatTextColor")` to a variable, then return that here.  Otherwise you will be running this function for every text label every time the screen renders, which is a big perf overhead.